src/Styleguide/Pages/Artwork/Sidebar/SizeInfo.tsx

```diff
@@ -3,7 +3,7 @@ import styled from "styled-components"
 import { Serif } from "@artsy/palette"
 import { themeGet } from "styled-system"
 
-interface SizeInfoProps {
+export interface SizeInfoProps {
```

Comment:
Don't need to export those any more. This now is just for testing those local small parts. The static page should use data from fixtures.